cscs-checks/apps/cp2k/cp2k_check.py

```diff
@@ -14,15 +14,17 @@ def __init__(self):
         self.executable = 'cp2k.psmp'
         self.executable_opts = ['H2O-256.inp']
 
-        energy = sn.extractsingle(r'\s+ENERGY\| Total FORCE_EVAL \( QS \) '
-                                  r'energy \(a\.u\.\):\s+(?P<energy>\S+)',
-                                  self.stdout, 'energy', float, item=-1)
+        energy = sn.extractsingle(
+            r'\s+ENERGY\| Total FORCE_EVAL \( QS \) '
+            r'energy [\[\(]a\.u\.[\]\)]:\s+(?P<energy>\S+)',
```

Comment:
I think this can be written in a simpler way:

```suggestion
            r'energy \[a\.u\.\]:\s+(?P<energy>\S+)',
```

Reply:
Would this regex match the old CP2K 7.1 output `(a.u)` as well? I tried to match both outputs since CP2K 7.1 is the current default on Piz Daint (even if for only two more days, the check would still fail).

Reply:
Ok then, it can't be written more simply then :-)